fastlane/lanes/build.rb

```diff
@@ -394,6 +418,13 @@ def build_and_upload_prototype_build(scheme:, output_app_name:, appcenter_app_na
       dsym_path: lane_context[SharedValues::DSYM_OUTPUT_PATH]
     )
 
+    upload_gutenberg_sourcemaps(
+      sentry_project_slug: sentry_project_slug,
+      release_version: release_version_current,
+      build_version: build_number,
+      app_identifier: app_identifier
+    )
```

Comment:
This source map will be used for the alpha builds (i.e. installable builds in PRs). The app is determined by the `app_identifier` parameter.